File: C#/sys_i/sys_ticket/Model/CustomerDto.cs
#nullable disable
namespace SysAdmin.Models;

public class CustomerDto
{
    public string fullName { set; get; }
    public string aliases { set; get; }
    public string birthOfDay { set; get; }
    public string email { set; get; }
    public string phone { set; get; }
    public string customType { set; get; }
    public decimal amountTotal { set; get; }
    public int bag_go { set; get; }
    public int bag_back { set; get; }
    public Dictionary<int, string> seats_go { set; get; }
    public Dictionary<int, string> seats_go_child { set; get; }
    public Dictionary<int, string> seats_back { set; get; }
    public Dictionary<int, string> seats_back_child { set; get; }
}

File: java/api-common/src/main/java/org/tasc/booking/apiclient/dto/CustomerDto.java
package org.tasc.booking.apiclient.dto;

import lombok.Data;

import java.util.List;
import java.util.Map;

@Data
public class CustomerDto {
    public  String fullName;
    public String aliases;
    public String  birthOfDay;
    private String email;
    private String phone;
    private String customType;
    private double amountTotal;
    public int bag_go;
    public int bag_back;
    public Map<Integer,String>  seats_go;
    public Map<Integer,String>  seats_go_child;
    public Map<Integer,String> seats_back;
    public Map<Integer,String> seats_back_child;
}
